src/TranslationLoaderManager.php

```diff
@@ -43,4 +50,28 @@ protected function getTranslationsForTranslationLoaders(
             })
             ->toArray();
     }
+
+    /**
+     * Confirms that migration has been run and 
+     * current context has a valid DB connection.
+     */
+    protected function hasValidDbRequirements() {
+        $model = config('translation-loader.model');
+        $model = new $model;
+        if(!$model instanceof Model){
+            return false;
+        }
+
+        try {
+            DB::connection()->getPdo();
+        } catch (\Exception $e) {
```

Comment:
Could you import all namespace in your PR?

Reply:
Did it!